package.json

```
@@ -1,14 +1,14 @@
 {
-    "name": "minecraft-java-core",
-    "version": "3.10.4",
+    "name": "minecraft-java-core-azbetter",
```

I don't think you'll want to add this in the PR
```suggestion
    "name": "minecraft-java-core",
```